cooltools/api/expected.py

```diff
@@ -391,18 +365,16 @@ def make_diag_tables(
     return diag_tables
 
 
-def make_block_table(clr, regions1, regions2, clr_weight_name="weight", bad_bins=None):
+def make_block_table(clr, regions1, regions2, clr_weight_name="weight"):
     """
     Creates a table that characterizes a set of rectangular genomic blocks
     formed by combining regions from regions1 and regions2.
     For every block calculate its "area" in pixels ("n_total"), and calculate
     number of "valid" pixels in each block ("n_valid").
```

Comment:
Creates a table of total and valid pixels for a set of rectangular genomic blocks
    defined by regions1 and regions2.
    For every block calculate its "area" in pixels ("n_total"), and calculate
    number of "valid" pixels ("n_valid").
    Valid pixels exclude "bad" pixels, which are inferred from the balancing
    weight column `clr_weight_name`.